api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -371,6 +372,32 @@ def _get_offers_by_ids(
         rules_subquery = sa.null()
         min_max_prices_subquery = sa.null()
 
+    offer_event_dates = (
+        sa.select(
+            sa.case(
+                (
+                    sa.func.max(offers_models.Stock.beginningDatetime).cast(sa.Date).is_(None),
+                    postgresql.array([sa.null().cast(sa.Date)]),
```

Comment:
Suggestions : 
- Comme tu testes `row.offer_event_dates|length` dans le code jinja plus bas, je trouverais plus cohérent de renvoyer un tableau vide, plutôt qu'un seul élément `null`, dans le cas où il n'y a pas de date.
- De plus, ne serait-il pas pertinent d'ajouter une condition sur `isEvent` pour renvoyer ce tableau vide, afin de peut-être éviter de parcourir les stocks de toutes les offres sans date ?

```suggestion
                    sa.or_(
                        sa.not_(offers_models.Offer.isEvent),
                        sa.func.max(offers_models.Stock.beginningDatetime).cast(sa.Date).is_(None),
                    )
                    sa.cast(postgresql.array([]), postgresql.ARRAY(sa.Date)),
```